runtime(compiler): Introduce event-driven primitives for SpotBugs (#16258)
vim/vim: runtime/doc/quickfix.txt

			\ function('GenericPostCompilerCommand'),
	\ }

When "PostCompilerAction" is available, "PostCompilerActionExecutor" is also
supported.  Its value must be a Funcref pointing to a function that always

> declares an only parameter of type string and decides whether |:execute| can

Reviewer: ```suggestion
declares a single parameter of type string and decides whether |:execute| can
```
